### DISCUSSION TITLE
Facilitate editing of many Aside/block elements in a sequence (UX, editing performance)

### DISCUSSION THREAD

**Discussion by AnRei123** [Azure/api-management-developer-portal#2633] [General]
## Bug description

To edit the configurations of several sections/blocks elements or Aside elements for different contents, you need to repeat the following steps again and again.

To edit two Aside elements in a sequence, you need to perform the following steps:
1. Select the frame of an Aside element frame
2. Click on the Edit icon. The Aside edit dialog window opens.
3. View or change the settings.
4. Close the Edit dialog window.

5. Select the frame of another Aside element frame (often very uncomfortable - especially in case of add. collapse elements!!!)
6. Click on the Edit icon. The Aside edit dialog window opens.
7. View or change the settings.
8. Close the Edit dialog window.

The same or more steps apply for editing the section/block elements for two blocks in a sequence as you often need to click the arrow icon to be able to select the blue section/block frame.

If you need to change or view the setting for several elements in a sequence, these are many clicks and takes too much time.

## Expected behavior

To edit two Aside elements in a sequence, you should only need to perform the following steps:
1. Select the frame of an Aside element frame
2. Click on the Edit icon. The Aside edit dialog window opens.
3. View or change the settings of the first Aside element.
_-- Keep the Aside edit dialog window open as finding and selecting the Aside element is often very hard!!!---_
4. Click into any widget or an empty Aside element with the mouse pointer. The Aside edit dialog window reflects the Aside settings of the newly selected Aside element and the second Aside element is automatically highlighted with the red frame.
5. Click on the Edit icon. The Aside edit dialog window opens.
6. Close the Edit dialog window to end viewing/changing the settings of any newly selected Aside element on the page.

This would not only mean two steps less for any further segment selection, it is also facilitating the selection of a further aside element which is often very uncomfortable, especially in case of additional collapse elements.

**Comment by ghost** [comment]
@AnRei123, thank you for opening this issue. We will triage it within the next few business days.

**Comment by azaslonov** [comment]
Hi @AnRei123, I see, I guess we could add a gesture like clicking on multiple elements (probably of similar type) holding `shift` key, then apply changes to all of them using a single editor window. Would that work for you?

**Comment by AnRei123** [comment]
This would also fine for some cases, but would only solve the issue for applying the same settings to multiple elements. Still it would be hard to find the right frame and you would not be supported in facilitating reviewing the different settings of several aside or block settings in a sequence.

**Comment by azaslonov** [comment]
Yeah, I get you. Thanks for feedback.
